examples/with-docker/packages/jest-presets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "jest-presets",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
+    "jest"
+  ],
```

Comment:
This is a good compromise on the `files` usage that I consistently comment on.

Reply:
Eh I'll still pull it, I just missed this one.